docs/content/docs/internals/job_scheduling.md

```diff
@@ -95,3 +95,22 @@ For that reason, the execution of an ExecutionVertex is tracked in an {{< gh_lin
 {{< img src="/fig/state_machine.svg" alt="States and Transitions of Task Executions" width="50%" >}}
 
 {{< top >}}
+
+## Repeatable Resource Cleanup Strategy
+
+Once a job has reached a terminally global state of either finished, failed or cancelled, the
+resources associated with the job are then cleaned up. This is done via a repeatable
+resource cleanup strategy, in which failures to clean up a resource result retries separated by
+an exponentially increasing delay, within configured bounds.
```

Comment:
```
This is done via a repeatable
resource cleanup strategy, in which failures to clean up a resource result retries separated by
an exponentially increasing delay, within configured bounds.
```
I struggle to understand that sentence. 🤔  What about: "In case of cleanup failure, you are able to configure different ways of retrying the cleanup." (addressing the user directly seems to be desired in the documentation according to @infoverload